taxi/views.py

```diff
@@ -58,6 +75,19 @@
     model = Car
     paginate_by = 5
     queryset = Car.objects.select_related("manufacturer")
+    def get_context_data(self, *, object_list=None, **kwargs):
+        context = super(DriverListView, self).get_context_data(**kwargs)
```

Comment:
The `get_context_data` method in `CarListView` uses `DriverListView` in `super()`, which is incorrect. It should be `CarListView`.